3.0.0

samdark opened the pull request:
Refactored library to be more flexible.

## TODO

- [ ] Implement video extension
- [ ] Implement image extension

samdark committed:
Refactored library to be more flexible

samdark committed:
Work in progress on Google extensions

iamsaint committed:
Fixes: Image extension write (#61)

lav45 committed:
Removed unused PHP version from .travis.yml

evannoronha commented:
@samdark is there a reason this pull wasn't merged? We want to use this feature for our sitemap generation.

samdark commented:
It isn't finished.

evannoronha commented:
Do you plan on finishing it?

samdark commented:
Yes but it's frozen now due to lack of time.

iamsaint committed:
Replace separator to make Google happy (#68)

Google can't properly read sitemap files with `_` in their name

DavidGoodwin committed:
Add Psalm static analysis, fix issues (#76)

DavidGoodwin committed:
Fix unit tests (#75)

php7.4 - mime type seems to change to application/gzip rather than application/x-grip
file names not correct.

wolfgangzwiauer commented:
Hey - what is missing for this to be merged/released? How can we help?

samdark commented:
1. Mege conflicts.
2. Video extensions.
3. Image extensions.

You can fork this branch and do your own pull request.
